src/Common/Api/Parameter.php

```diff
@@ -285,6 +285,11 @@ private function hasCorrectType($userValue): bool
             return true;
         }
 
+        // allow string nulls
+        if ($this->type == 'string' && $userValue === null) {
```

Comment:
I think this is really inconsistent on OpenStack's part (what's new) but I don't think there's an easier way to do this.